css/app.css

```diff
@@ -35,3 +35,28 @@ gaia-list .destructive {
 a.navigation *, a.action *, a.edit * {
   pointer-events: none;
 }
+
+gaia-list span.group {
+  text-transform: capitalize;
+  font-size: 0.8em;
+  min-height: auto;
+
+  background: rgba(255, 255, 255, 0.35);
+  color: var(--text-color);
+}
+
+gaia-list span.group::before {
+  position: absolute;
+  top: 0;
+  left: -20px;
+  width: calc(100% + 20px);
+  height: 100%;
+  content: '';
+
+  background-color: var(--highlight-color);
+  opacity: 0.2;
+}
```

Comment:
so I tried something, and it seems to work well and is (IMO) less hacky and so is more future proof:
```css
gaia-list span.group {
  ...
  background: linear-gradient(rgba(255, 255, 255, 0.7), rgba(255, 255, 255, 0.7)), var(--highlight-color);
  ...
}

gaia-list span.group::before {
  display: none;
}
```

The idea is to use a "solid" gradient with an alpha layer that will lighten the color you chose.

The best way forward will be to have a proper color for this, but I suppose this is not for now.
Tell me what you think.